### PR TITLE
Tj/display goodbye message

### PR DESCRIPTION
**Pull request by ghost**
Review and Merge, please!

### CODE REVIEW

**Comment by ghost** [comment]
Don not forget to disable this: LOGOUT_REDIRECT_URL = 'lmn:homepage' in your settings.py

**Comment by Di30an** [comment]
Seems there is still an issue with the website redirecting to fast. The displayLength does not seem to be the issue, rather the location.href that is causing it to automatically redirect. I was able to get the code to run locally by adding :
image

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,7 @@
 *.pyc
 __pycache__
-venv
\ No newline at end of file
+lmnEnv
```
Comment (molly-clabots) [inline .gitignore:4]: There are a couple conflicts to resolve. One of them is with this file. If you don't know where to do that there is a button on the conversations page.

**Review by molly-clabots — APPROVED** [review]
Looks nice TJ. I like the styles you choose.

**Pull request merged**